docs/0-intended-use/index.md

```diff
@@ -278,11 +197,11 @@ configurable alerts and warnings can provide the appropriate notification.
 ## Part of the Body / Type of Tissue Interacted With
 
 The device is stand-alone software. It receives input from the user and outputs
```

Comment:
What input does it receive from the user (if the data comes over the network)?
It also receives input from the network if the DICOM DIMSE/wire protocol is in use

Reply:
Fixed in d9fc69e62dcf8ed6eca258b7beb0e20c32bc031e